gpu4pyscf/lib/gint/bpcache.cu

```diff
@@ -53,6 +53,7 @@ void GINTdel_basis_prod(BasisProdCache **pbp)
     
     if (bpcache->aexyz != NULL) {
         free(bpcache->aexyz);
+        free(bpcache->h_bas_coords);
```

Comment:
bas_coords on host can be passed into CPU kernel directly. No need to save it in bpcache.